presto-hive/src/test/java/com/facebook/presto/hive/TestHiveHistoryBasedStatsTracking.java

```diff
@@ -350,6 +350,39 @@ public void testHistoryBasedStatsCalculatorCTE()
         assertPlan(cteMaterialization, sql, anyTree(node(ProjectNode.class, anyTree(any())).withOutputRowCount(3)));
     }
 
+    @Test
+    public void testStrategy()
```

Comment:
The test name is rather vague and doesn't convey what its testing

Reply:
@jaystarshot Thanks for your comment. I have changed the test function's name to make it more clear meaning, please take a look when convenient.